wsd/SenderQueue.hpp

```diff
@@ -90,11 +90,24 @@ class SenderQueue final
         {
             // Remove previous identical tile, if any, and use most recent (incoming).
             const TileDesc newTile = TileDesc::parse(item->firstLine());
+            uint32_t newTilePosHash = newTile.equalityHash();
+            // store a hash of position for this tile.
+            item->setHash(newTilePosHash);
```

Comment:
"Item" is only ever a std::shared_ptr<Message> so pointee of const Item& item can be modified, and this is the only place something can get put into the queue.

Not entirely sure that SenderQueue makes sense as a template where only one type could work, but that's not particularly relevant to the change.